app/src/test/java/org/mozilla/fenix/home/recentbookmarks/RecentBookmarksFeatureTest.kt

```diff
@@ -86,23 +81,4 @@ class RecentBookmarksFeatureTest {
                 assertEquals(listOf(bookmark), it.recentBookmarks)
             }
         }
-
-    @Test
-    fun `WHEN the feature is destroyed THEN the job is cancelled`() {
```

Comment:
Removed this test as well because it's testing calls that we make ourself, but is also flaky because it assumes the job is non-null at the assertion point.